src/plugin/regions/region.js

```diff
@@ -606,13 +606,33 @@ export class Region {
         });
     }
 
+    /**
+     * @example
+     * onResize(-5, 'start') // Moves the start point 5 seconds back
+     * onResize(0.5, 'end') // Moves the end point 0.5 seconds forward
+     *
+     * @param {number} delta How much to add or subtract, given in seconds
+     * @param {string} direction 'start 'or 'end'
+     */
     onResize(delta, direction) {
         if (direction === 'start') {
+            // Check if changing the start by the given delta would result in the region being smaller than minLength
+            // Ignore cases where we are making the region wider rather than shrinking it
+            if (delta > 0 && Math.abs(this.end - (this.start + delta) < this.minLength)) {
+                return;
+            }
+
             this.update({
                 start: Math.min(this.start + delta, this.end),
                 end: Math.max(this.start + delta, this.end)
             });
         } else {
+            // Check if changing the end by the given delta would result in the region being smaller than minLength
+            // Ignore cases where we are making the region wider rather than shrinking it
+            if (delta < 0 && Math.abs((this.end + delta) - this.start) < this.minLength) {
```

Comment:
same question about `Math.abs`.  Also curious why we have to check for `delta > 0` and `delta < 0`; the checks are sorta fine, but was there a specific reason to put them in?

Reply:
Since minLength isn't checked everywhere, it's actually possible to manually call this.update() and set the region to a size smaller than minLength. The delta check makes sure that if the area is too small, you can still make it bigger, but not shrink it further when dragging it with the mouse. A better fix would be to make sure the region can never be smaller than minLength, but I'm not familiar enough with this plugin to make that change.